src/contentScript/contentScript.css

```
@@ -44,6 +44,14 @@
   box-shadow: 0 4px 0.25rem rgba(0, 0, 0, 0.1);
   z-index: 100;
   pointer-events: none;
+
+  width: 300px;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)